Dockerfile

```diff
@@ -61,6 +61,32 @@ RUN (cd /lib/systemd/system/sysinit.target.wants && for i in *; do [ $i == syste
 ## Remove any existing configurations
 RUN rm -f /etc/httpd/conf.d/*
 
+## For Ruby
+ENV TERM=xterm \
+    LANG=en_US.UTF-8 \
+    RUBY_GEMS_ROOT=/opt/rubies/ruby-2.3.1/lib/ruby/gems/2.3.0 \
+    PATH=$PATH:/opt/rubies/ruby-2.3.1/bin
+
+# Install repos
+RUN yum -y install https://dl.fedoraproject.org/pub/epel/epel-release-latest-7.noarch.rpm && \
+    curl -sL https://copr.fedorainfracloud.org/coprs/postmodern/ruby-install/repo/fedora-25/postmodern-ruby-install-fedora-25.repo -o /etc/yum.repos.d/ruby-install.repo && \
+    sed -i 's/\$releasever/25/g' /etc/yum.repos.d/ruby-install.repo
+
+# Install ruby-install and make
+RUN yum -y install --setopt=tsflags=nodocs ruby-install make
+
+# Install Ruby 2.3.1
+RUN ruby-install ruby 2.3.1 -- --disable-install-doc && rm -rf /usr/local/src/* && yum clean all
+
+## Build Auth-Api
+ENV HTTPD_AUTH_API_SERVICE_DIRECTORY=/opt/rh/auth-api
```

Comment:
This shouldn't be rh specific. I think that comment got lost in the other file.

Did we come to a consensus about what the path should be? @bdunne @Fryguy

Reply:
We decided `/opt/auth-api`

Reply:
yep, 👍